src/main/java/org/apache/commons/collections4/bidimap/DualHashBidiMap.java

```diff
@@ -22,6 +22,7 @@
 import java.io.Serializable;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.LinkedMap;
```

Comment:
`s/LinkedMap/LinkedHashMap`

Make sure you read `CONTRIBUTING.md` too as it contains useful information for contributing code.

Reply:
Yes, thanks